### PR TITLE
Applied corrections discussed in #1037

### PR DESCRIPTION
**Pull request by gozora**
Spelling corrections
BORG_* variables renamed to BORGBACKUP_* to avoid collisions with Borg.
When ReaR uses Borg as back end, it defaults to "none" compression to be aligned with Borg defaults.

### CODE REVIEW

**Comment by jsmeix** [comment]
I have a question:
Now there is is default.conf
<pre>
BORGBACKUP_COMPRESSION="none"
</pre>
I assume this enforces "none" compression for rear
regardless what the users may have set in its Borg
configuration (I assume the user can configure
a compression that Borg should use), cf
https://github.com/rear/rear/pull/1037#discussion_r84023889
where I assumed that with an empty
BORG_COMPRESSION=""
the Borg default gets used.

Bottom line: I wonder if
<pre>
BORGBACKUP_COMPRESSION=""
</pre>
would be more in compliance with Borg?

**Comment by gozora** [comment]
@jsmeix , exactly _none_ compression is enforced now. I guess this is not right thing to do :-(.
`BORGBACKUP_COMPRESSION=""` will not however work as value is appended like this
`borg create --compression <BORGBACKUP_COMPRESSION> ..` so it would throw syntax error.

I'll think about some way how to solve this.

Thanks for checking!

**Comment by gozora** [comment]
ehm, not sure if it is clear from above text, but user can change compression setting e.g. `BORGBACKUP_COMPRESSION="zlib,9"` in _local.conf_

**Comment by jsmeix** [comment]
Of course the user can specify what he wants in local.conf
but my point and Thomas Waldmann's point in
https://github.com/rear/rear/pull/1037#discussion_r83887959
is that when the user does not specify anything,
then rear should use Borg in its default mode.

In other words:
When the user does not specify anything in rear,
rear should not specify something special for Borg.

**Comment by gozora** [comment]
Ok, I'll correct it.
